.github/workflows/coverage_reporting.yml

```diff
@@ -53,8 +53,9 @@ jobs:
         uses: codecov/codecov-action@v4
         with:
           fail_ci_if_error: true
-          token: ${{ secrets.CODECOV_TOKEN }}
           verbose: true
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Im curious, how will this help? :smile:

Reply:
In PR 979, there were consistent GitHub rate limit errors.  Upon investigation, changing this resolved the issue.

A reviewer there requested this change be split off.

Historically, we've seen in other projects where alternate implementations result in huge improvements in reliability.  This PR builds off our prior experience with codecov.